origin.spec

```diff
@@ -311,6 +311,18 @@ install -m 0644 contrib/tuned/man/tuned-profiles-origin-node.7 %{buildroot}%{_ma
 
 mkdir -p %{buildroot}%{_sharedstatedir}/origin
 
+install -d %{buildroot}/usr/local/bin/
+
+# Install master scripts
+install -p -m 0755 images/origin/system-container/origin-system-container-wrapper.sh %{buildroot}/usr/local/bin/
```

Comment:
@giuseppe and @tdawson would want general confirmation that the changes to system container stuff here is kosher and that we are OK to be putting this into the RPMs